tezos_interop/webpack.config.js

```diff
@@ -1,12 +1,13 @@
-module.exports = env => ({
+module.exports = (env) => ({
   entry: {
-    run_entrypoint: './run_entrypoint.js',
-    listen_transactions: "./listen_transactions.js"
+    run_entrypoint: "./run_entrypoint.js",
+    fetch_validators: "./fetch_validators.js",
+    listen_transactions: "./listen_transactions.js",
   },
   output: {
-    filename: '[name].bundle.js',
+    filename: "[name].bundle.js",
     path: __dirname,
   },
-  mode: env.dev ? 'development' : 'production',
-  target: 'node'
+  mode: env.dev ? "development" : "production",
```

Comment:
I have opened https://github.com/marigold-dev/sidechain/pull/159 to prevent noise like this in future